app/assets/stylesheets/content/work_packages/resizer/resizer.sass

```diff
@@ -1,6 +1,7 @@
 .work-packages--resizer
   position: absolute
   top: 50%
+  left: -2px
```

Comment:
Are you aware that you made the left padding of the right side of the vertical line wider? ... Actually that might be necessary to let the drag handle icon not interfere with bullet points.

Reply:
Exactly. The resizer would overlap the bullet points. That is why I increased the padding.